src/macaron/slsa_analyzer/checks/check_result.py

```diff
@@ -37,7 +39,7 @@ class CheckResult(TypedDict):
     justification: list[str | dict[str, str]]
     # human_readable_justification: str
     # result_values: dict[str, str | float | int] | list[dict[str, str | float | int]]
-    result_tables: list[DeclarativeBase | Table]
+    result_tables: list[DeclarativeBase | Table | Expectation]
```

Comment:
`Table` is a left over from an old implementation and can be removed too.

Reply:
Removed